src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/FileWriter.java

```diff
@@ -42,7 +42,7 @@ public class FileWriter implements Closeable {
     private Timer timer;
     private final Consumer<SourceFile> onRollCallback;
     private final Function<Long, String> getFilePath;
-    private OutputStream outputStream;
+    private GZIPOutputStream outputStream;

```

Comment:
@ohadbitt can you check if the changes in file to address some more comments look okay